src/main/webapp/WEB-INF/fenixedu-academic/accounting/events-multiple-payments-confirm.jsp

```diff
@@ -29,6 +29,15 @@
 
 <script type="text/javascript">
 
+    window.onbeforeunload = function() {
+        // this message is not shown anymore when using the most recent browsers.
```

Comment:
This message is only shown in Internet Explorer 🤣
https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#Browser_compatibility